src/Integrations/Insightly/Resources/OpportunityResource.php

```diff
@@ -46,6 +46,8 @@ public function create(Opportunity $opportunity): Id
 
         $this->updateStage($id, $opportunity->getStage());
 
+        $this->linkContact($id, $opportunity->getContactId());
+
```

Comment:
Nit:  Maybe we can pass the contact as a 2nd argument and use a name like `createWithContact()`? So that it's a tiny bit more explicit.

Reply:
Good point! I'm gonna keep this for now but I created a follow-up task to check if the `ContactId` needs to be contained and required inside a `Project` and `Opportunity`
See: https://jira.uitdatabank.be/browse/PROJ-187